### DISCUSSION TITLE
Documentation around cache is unclear

### DISCUSSION THREAD

**Discussion by hpachy** [urql-graphql/urql#3084] [Q&A]
I would like to speak about the documentation which is for me very painful to understand... So I'm wondering if I m the only one who's struggling to understand if it's the case ignore and close this issue...

So the thing I really want to do in the Urql is a better documentation about the cache and the implementation / modification of it, with less abstract exemple. I found the documentation very blurry about the utilisation of every function of the cache, the only one I could implemented was the `cache.invalide` but for the other even after few days of testing still nothing usable for my app

## Summary

I proposing to write a documentation we explicite exemple, to make the cache utilisation more easier for beginners like me, seeing some exemple that s can be reproduce can be very help full to understand the cache methodes

## Proposed Solution

Sadly I don't have the solution because even days struggling on the docs I still not found the way to update the cache, I just found the way to use `cache.invalide` but the current probleme I'm facing is to create a new element nested to an object in the cache

**Comment by JoviDeCroock** [comment]
I'm sorry but... this isn't an RFC this is a complaint about the docs 😅 it however would be very valuable feedback if you could point out all the parts of the docs that aren't clear and why.

**Comment by kitten** [comment]
If you have more general questions, feel free to drop by our Discord: https://github.com/urql-graphql/urql/issues/3016
I'm also happy to discuss what you'd like to see from the docs there.
In general, if you're looking for runnable examples, we've got for instance this one, which should be modifiable against a testing API:
- https://github.com/urql-graphql/urql/blob/main/examples/with-graphcache-updates/src/client.js

**Comment by hpachy** [comment]
@JoviDeCroock So sorry...
@kitten I'm gonna try the link you gave me and if I'm still struggling I ll ask on the discord for help...

and if I figured out what was wrong for my comprehension I ll create a new issue to propose a solution

**Comment by hpachy** [comment]
hello ! so thx to the discord I figured out the solution to my problem so I drop it to help and when I have time I ll make an issue to add more exemple to the doc for a better understanding for beginners :) 
https://discord.com/channels/1082378892523864074/1085940813499998288

Thx for the Urql teams once again

**Comment by kitten** [comment]
Converting this to a discussion for now, since it's not actionable ✌️
